poetry.lock

```diff
@@ -1327,7 +1327,7 @@ description = "Core utilities for Python packages"
 name = "packaging"
 optional = false
 python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
-version = "20.1"
```

Comment:
Is there any need for the poetry.lock changes? All that changed in pyproject.toml is the order of dependency declaration